Exit challenger service (#15)
leapdao/challenge-services: README.md

## challenge-services

> A JavaScript tools to ensure validity of a LeapDAO plasma chain.

### Content

This repository contains tools to provide MVP Plasma challenge games.

> Each tool has it's own subdirectory with own README file and the whole codebase to run tools independently of each over or together, depends on your needs.

Reviewer: ```suggestion
Each tool is in its own subdirectory with README file. Tools can be run separately or together, depending on your needs.
```
